cypress/e2e/functional/auth-spec.cy.js

```diff
@@ -1,7 +1,18 @@
+import 'cypress-axe';
 describe('Login', () => {
-  it('loads login page', () => {
+  before(() => {
     cy.visit('/login');
+    cy.injectAxe();
+  });
+
+  it('loads login page', () => {
     cy.get('h1').should('contain', 'Sign in to the DARTS Portal');
+    cy.checkA11y(null, {
+      runOnly: {
+        type: 'tag',
+        values: ['wcag22aa'],
+      },
+    });
```

Comment:
I'm wondering if this config should be kept somewhere centrally to avoid having to define it every time? Either that or wrap the command so we can set it in one place?

Reply:
Can maybe add in `commands.js`?

Reply:
Added